dcat/index.html

```diff
@@ -264,24 +264,20 @@ <h3>DCAT scope</h3>
         A <b>dataset</b> in DCAT is defined as a "collection of data, published or curated by a single agent, and available for access or download in one or more serializations or formats".
         A dataset is a conceptual entity, and can be represented by one or more <b>distributions</b> that serialize the dataset for transfer.
         Distributions of a dataset can be provided via <b>data services</b>.
-        Detailed properties for a data service API are out of the scope of this version of DCAT.
     </p>
 
     <p>
-      A <b>data service</b> in DCAT is a collection of operations or API which provides access to data.
-      An interactive interface may provide access to the API operations.
```

Comment:
Is this not correct? It does not say that the interactive interface is the service, merely that an interactive interface might be available to provide access (i.e. an application that uses the API).

Reply:
While technically true, it confuses the issue. We are defining a DataService as an API. If we kept this, we would need to explain that an exposed API itself would need to be the thing being catalogued.